### PR TITLE
Updates after post-mortem

### PR DESCRIPTION
**Pull request by biglovisa**
- Add instructions on how to build/test the package locally 
- Add a PR template checklist 
- Run the tests before publishing the gem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,3 +138,57 @@ patch.preferences.deletePreference(preferenceId);
...
+
+```
+$ node
+> const Patch = require('@patch-technology/patch')
+> Patch.default('<PATCH_API_KEY>').projects.retrieveProjects().then((response) => console.log(response))
+```
+
+### Run the specs
+
+Before running the tests, make sure you set the API key!
```
Comment (thdaraujo) [inline README.md:184]:
```suggestion
Before running the tests, make sure you set the test API key! (please use test API keys and not production ones, they usually start with `test_`)
```

File: README.md [hunk 2]
```diff
@@ -138,3 +138,57 @@ patch.preferences.deletePreference(preferenceId);
...
+
+```
+$ export SANDBOX_API_KEY=<PATCH_API_KEY>
```
Comment (thdaraujo) [inline README.md:187]:
```suggestion
$ export SANDBOX_API_KEY=<PATCH_TEST_API_KEY>
```

**Review by thdaraujo — APPROVED** [review]
that's good, maybe remind the user to use test api keys as well.

Maybe we should think about some form of check for that, it's not fun to use production keys on your local tests.

**Comment by biglovisa** [comment]
@thdaraujo very good points - thanks for the suggestions! 🙏

**Pull request merged**